inc/item_operatingsysteminjection.class.php

```diff
@@ -92,6 +92,25 @@ function addOrUpdateObject($values = [], $options = []) {
       return $lib->getInjectionResults();
    }
 
+
+   /**
+    * @param PluginDatainjectionInjectionInterface $injectionClass
+    * @param array $values
+    * @param array $options
+   **/
+  function customDataAlreadyInDB($injectionClass, $values, $options) {
+      $item_operatingsystem = new Item_OperatingSystem();
+      if (!$item_operatingsystem->getFromDBByCrit([
+         'itemtype' => $values['itemtype'],
+         'items_id' => $values['items_id'],
+      ])) {
```

Comment:
`getFromDBByCrit` will return false if there are already mutiple OS entries, so the problem will not be solved for people that already used the plugin to inject multiple OS on a computer.

Reply:
last commit handle this case (like ```Domain``` from native inventory)